Automattic/jetpack
# My Jetpack: Add new `Text` component at `js-packages/components`

It creates a new component that will keep consistency across our texts, following all tokens proposed by Design at: PbJqbW-8t-p2

Usage example drafted on #23311 

#### Changes proposed in this Pull Request:

* Creates new `Text` component

#### Jetpack product discussion


PbJqbW-8t-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Storybook`.
* Goes to `Components / Text`.
* Interact with `Controls` tab, you should see all variants.

#### Demo

![Screen Shot 2022-03-14 at 18 52 23](https://user-images.githubusercontent.com/1663717/158268577-b6466ed5-eeb2-4119-84c6-4b0377230b3d.png)
